schedule_scaling/schedule_scaling.py

```diff
@@ -44,64 +44,80 @@ def get_kube_api():
     return api
 
 
-def deployments_to_scale():
+def get_resource_to_scale(resource_type="deployment"):
     '''
-    Getting the deployments configured for schedule scaling...
+    Getting the resource configured for schedule scaling...
     '''
+    if resource_type not in ['deployment', 'hpa']:
```

Comment:
We should remove the support for deployments. The problem with deployments is that if we do nightly scale down on deployments that dont auto scale, in-case if there is a need for more capacity, nothing would happen. Enforcing this via HPAs will make sure that in case the service needs more capacity at night, it can autoscale